using/solving-exercises/README.md

```diff
@@ -0,0 +1,21 @@
+# Solving Exercises
+
+The primary way in which Exercism helps you learn is by solving exercises.
+
+There are two ways to solve exercises on Exercism:
+
+1. [Online via the website's in-browser editor](./online.md)
+2. [On your local machine via the CLI](./locally)
+
+## Which option should I choose?
+
+There are different trade-offs between using the online editor or CLI:
+
+|            | Online editor                          | CLI                             |
+| ---------- | -------------------------------------- | ------------------------------- |
+| Installing | No                                     | Yes (e.g. SDK)                  |
+| Editing    | Browser                                | Your preferred IDE              |
```

Comment:
```suggestion
| Editing    | Browser                                | Your preferred text editor      |
```
Better to be more general here? Not everyone uses an IDE, but an IDE contains a text editor.

Reply:
Or maybe just "editor"?